clc/modules/block-storage/src/main/java/com/eucalyptus/blockstorage/DASManager.java

```diff
@@ -787,6 +787,46 @@ public boolean shouldExportOnReload(LVMVolumeInfo lvmVolumeInfo) {
       return returnValues;
     }
 
+    protected String enableLogicalVolume(String lvName) throws EucalyptusCloudException {
+      
+      EucalyptusCloudException ex = null;
+      String result = null;
+
+      // 1st timeout is 10 ms, 2nd is 40. Last (9th) is 2.56 sec. Total 3.41 sec.
+      int attempt = 1;
+      final int max_attempts = 9;
+      int timeout = 10; //ms
+      final int timeout_multiplier = 4;
+
+      do {
+        try {
+          result = LVMWrapper.enableLogicalVolume(lvName);
+          if (attempt > 1) {
+            LOG.info("Enabling " + lvName + "succeeded on retry.");
```

Comment:
That was intentional, only logging an INFO message if a retry had to be attempted. We can also log each attempt as you show, but I didn't think it was needed on DEBUG. We already have DEBUG messages around enabling volumes. Will add your Enabling log message on TRACE level instead if that's OK.